l10n_es_ticketbai_global_discount/__manifest__.py

```diff
@@ -0,0 +1,35 @@
+###############################################################################
+#
+#    SDi Soluciones Informáticas
+#    Copyright (C) 2023-Today SDi Soluciones Informáticas <www.sdi.es>
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+###############################################################################
+{
+    'name': 'Global discounts ticketbai',
+    'summary': '''This module sends global discounts as details for ticketbai
+    ''',
+    'author': 'Sergio Lop Sanz, SDi Soluciones Informáticas',
```

Comment:
```suggestion
#    SDi Digital Group
#    Copyright (C) 2023-Today SDi Digital Group <www.sdi.es>
#
#    This program is free software: you can redistribute it and/or modify
#    it under the terms of the GNU Affero General Public License as
#    published by the Free Software Foundation, either version 3 of the
#    License, or (at your option) any later version.
#
#    This program is distributed in the hope that it will be useful,
#    but WITHOUT ANY WARRANTY; without even the implied warranty of
#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
#    GNU Affero General Public License for more details.
#
#    You should have received a copy of the GNU Affero General Public License
#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
#
###############################################################################
{
    'name': 'Global discounts ticketbai',
    'summary': '''This module sends global discounts as details for ticketbai
    ''',
    'author': 'Sergio Lop Sanz, SDi Digital Group',
```